lib/new_relic/agent/instrumentation/excon.rb

```diff
@@ -64,10 +64,12 @@ def install_middleware_excon_instrumentation
     end
   end
 
+  # TODO: Remove in 8.0.0 release
```

Comment:
Since this is going into the 8.0.0 release, can we go ahead and remove this?

Reply:
Absolutely!